assets/css/learning-mode.scss

```diff
@@ -4,12 +4,11 @@
 @import 'sensei-course-theme/buttons';
 @import 'sensei-course-theme/quiz';
 @import 'sensei-course-theme/notices';
-@import 'sensei-theme-blocks';
```

Comment:
These contain frontend styles so it should still be included, either in the learning-mode bundle, or as a dependency. 

Oh looks like right now it's [added as a style for the Course Navigation](https://github.com/Automattic/sensei/blob/481dcaa33bd6708916a3f6b131e1a813fa4f25aa/includes/blocks/course-theme/class-course-navigation.php#L64) block, which masks the problem as it's loaded as long as that block is used.

Reply:
Changed here for now: https://github.com/Automattic/sensei/pull/4806/commits/de3d2109d5dd775fcf02b18a7a89c0f394c1ed6a

Reply:
I investigated a little more and realized that we weren't registering the blocks properly in the backend. It was fixed here: https://github.com/Automattic/sensei/pull/4806/commits/21d8271b6cdfa399910182204308a9be9c6204c1